File: typescript/test/GamePointCalculator.test.ts
import { RollPoint, SPARE, STRIKE } from '../src/RollPoint';
import { evaluateGame } from '../src/GamePointCalculator';

import {describe, expect, test} from '@jest/globals';

// there are 20 rolls
test('The most regular player: 2 pins for roll', () => {
  var game = Array<RollPoint>(20).fill(RollPoint.of(2));
  const ans = evaluateGame(game);
  expect(ans).toBe(2 * 20);
});

File: java/src/test/java/it/leroymerlin/GamePointCalculatorTest.java
package it.leroymerlin;

import org.junit.jupiter.api.Disabled;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

import java.util.Arrays;

import static org.hamcrest.MatcherAssert.assertThat;
import static org.hamcrest.Matchers.is;

class GamePointCalculatorTest {

    @Test
    @DisplayName("The most regular player: 2 pins for roll")
    // there are 20 rolls
    public void allTheSamePinsForRole() {

        RollPoint[] rollPoints = new RollPoint[20];
        Arrays.fill(rollPoints, RollPoint.of(2));
        int result = GamePointCalculator.evaluateGame(rollPoints);
        assertThat(result, is(2 * 20));

    }

}